plugins/explore/src/components/ToolCard/ToolCard.tsx

```diff
@@ -14,19 +14,22 @@
  * limitations under the License.
  */
 
-import React from 'react';
-import classNames from 'classnames';
+import { BackstageTheme } from '@backstage/theme';
 import {
   Button,
   Card,
   CardActions,
   CardContent,
   CardMedia,
   Chip,
-  Typography,
   makeStyles,
+  Typography,
 } from '@material-ui/core';
-import { BackstageTheme } from '@backstage/theme';
+import classNames from 'classnames';
+import React from 'react';
+import { CardData } from './types';
+
+// TODO: Align styling between Domain and ToolCard
```

Comment:
I really like the tool card styling (tags at the bottom, the ability to add a picture alter) but I prefer to go with the build in style from storybook for both cards. Maybe we improve the `ItemCard`?

Reply:
Yeah we should unify this later and make them great.